src/quo2/components/profile/select_profile/view.cljs

```diff
@@ -0,0 +1,53 @@
+(ns quo2.components.profile.select-profile.view
+  (:require
+    [quo2.components.profile.select-profile.style :as style]
+    [react-native.core :as rn]
+    [quo2.components.markdown.text :as text]
+    [quo2.components.avatars.user-avatar :as user-avatar]
+    [reagent.core :as reagent]))
+
+(defn- on-change-handler
+  [selected? on-change]
+  (swap! selected? not)
+  (when on-change (on-change @selected?)))
+
+(defn view
+  "Options
+   - `default-selected?` Default selected state
+   - `selected?` Selected state
+   - `name` Full name
+   - `profile-picture` Profile picture
+   - `customization-color` Customization color
```

Comment:
not directly related to this p.r:
@Parveshdhull here we are using prop name `customization-color`, same in "src/quo2/components/buttons/dynamic_button.cljs" and some other places.

however I see in "src/quo2/components/settings/accounts/view.cljs" we are using `custom-color`, and I can imagine there might be some other variants. (`color` etc)

I think we should update the best practices so that whenever a component is using this set of keys for custom colors that they use the prop `customization-color` for consistency. 
WDYT?

Reply:
sounds good to me

Reply:
deja vu :) https://github.com/status-im/status-mobile/pull/14127#discussion_r991928328